mindsdb/integrations/handlers/web_handler/urlcrawl_helpers.py

```diff
@@ -306,6 +336,7 @@ def dict_to_dataframe(dict_of_dicts, columns_to_ignore=None, index_name=None):
 
 if __name__ == "__main__":
```

Comment:
I implemented the tests using `unittest` first. They're in `pytest` now and safely removable.

Reply:
Since it's a helper, it can easily be removed as it never gets executed sequentially. It's also testing the `pdf_to_markdown` function which is not right. Now, we have tests. :)